src/utils/helpers.js

```diff
@@ -0,0 +1,30 @@
+import { getDaysBetweenDates } from './dates';
+
+export function getDaysSinceLastPurchase(item) {
+	const dateLastPurchasedJavaScriptObject = item.dateLastPurchased
+		? item.dateLastPurchased.toDate()
+		: item.dateCreated.toDate();
+
+	return getDaysBetweenDates(new Date(), dateLastPurchasedJavaScriptObject);
+}
+
+export function getIndicator(item) {
+	const daysSinceLastPurchase = getDaysSinceLastPurchase(item);
+
+	const daysUntilNextPurchase = getDaysBetweenDates(
+		item.dateNextPurchased.toDate(),
+		new Date(),
+	);
+
+	if (daysSinceLastPurchase > 60) {
+		return 'Inactive';
+	} else if (daysSinceLastPurchase > 30 && daysSinceLastPurchase < 60) {
+		return 'Overdue';
```

Comment:
i believe when `dateNextPurchased`has passed is how an item becomes Overdue.

Reply:
Thanks for spotting this! Should be working correctly now

Reply:
No problem!